### PR TITLE
add alpha text inside logo and fix styles

### PR DESCRIPTION
**Pull request by MixailE**

### CODE REVIEW

File: src/components/Text.tsx [hunk 1]
```diff
@@ -132,11 +132,19 @@ export function CardDescription({ children }: ChildrenProp) {
...
 )
 export function LogoText({ children }: ChildrenProp) {
   return <span className={logoText}>{children}</span>
 }
+const logoVersionText = classnames(
+  textColor('text-primary-semi-dimmed'),
+  fontWeight('font-bold'),
+  fontSize('text-xs')
+)
+export function LogoVersion({ children }: ChildrenProp) {
```
Comment (smyzin) [inline src/components/Text.tsx:145]: Maybe `LogoSubText` would be a better name for that sort of text 🤔

File: src/components/navbar/Navbar.tsx [hunk 2]
```diff
@@ -36,10 +37,12 @@ const navbar = (visible?: boolean, withoutWallet?: boolean) =>
 const logoContainer = classnames(
   display('inline-flex'),
   alignItems('items-center'),
-  space('space-x-2'),
+  space('space-x-4'),
   margin('mt-2')
 )
 
+const logoWithVersion = classnames(display('flex'), flexDirection('flex-col'))
```
Comment (smyzin) [inline src/components/navbar/Navbar.tsx:44]: There is no any space between them? If so, okay than )
But recheck it please 🙏

**Review by smyzin — APPROVED** [review]
Left tiny cosmetic comments but looks good

**Review by T-Damer — APPROVED** [review]
Lgtm

**Pull request merged**